lib/Bio/EnsEMBL/DataCheck/Checks/PhenotypeMultipleSeqRegions.pm

```diff
@@ -59,8 +59,11 @@ sub skip_tests {
   # because the query runs quickly.
   if ($pass) {
     my $mca = $dna_dba->get_adaptor("MetaContainer");
-    my $division = $mca->get_division;
+    if (! defined $mca) {
+      die("no mca")  # Make a better die message!
```

Comment:
Can the die message be updated and the comment removed.

Reply:
Yes, thanks. I updated it below.